src/test/java/LoginTests.java

```diff
@@ -8,25 +8,25 @@ public class LoginTests extends BaseTest {
     @Test
     public void LoginTest() throws InterruptedException {
         HomePage homePage = new LoginPage(driver)
-                .enterEmail("demo@class.com")
-                .enterPassword("te$t$tudent")
+                .enterEmail("uraganchic2012@gmail.com")
+                .enterPassword("Testpro1416!")
                 .submit();
         Assert.assertTrue(homePage.isLoggedIn());
 
 //        Just checking :)
 //        Assert.assertEquals(driver.getCurrentUrl(), "Fail please, so we see the properties: " + parameters);
     }
 
-    @Test
-    public void LoginInvalidPasswordTest() throws InterruptedException {
-        LoginPage loginPage = new LoginPage(driver);
-        loginPage
-                .enterEmail("demo@class.com")
-                .enterPassword("incorrectpassword")
-                .submit();
-        Assert.assertTrue(loginPage.isErrorShown(), "No error shown on Login page");
+//    @Test
+//    public void LoginInvalidPasswordTest() throws InterruptedException {
+//        LoginPage loginPage = new LoginPage(driver);
+//        loginPage
+//                .enterEmail("demo@class.com")
+//                .enterPassword("incorrectpassword")
+//                .submit();
+//        Assert.assertTrue(loginPage.isErrorShown(), "No error shown on Login page");
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.